### ISSUE TITLE
[Suggestion] Linux builds

### ISSUE THREAD

**Issue by i-am-called-glitchy** [Nokorpo/LibreAim#44]
bottom text

**Comment by antimundo** [comment]
Yeah I would like to add Linux builds, probably I should set up some kind of CI/CD environment to make both Windows and Linux builds at the same time
